modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -363,6 +363,17 @@ neo4j-admin database import full @/path/to/your/<args-filename> mydb
 For more information, see link:https://picocli.info/#AtFiles[Picocli -> AtFiles] official documentation.
 ====
 
+[NOTE]
+.Using both a multi-value option and a positional parameter
+====
+When using both a multi-value option, such as `--nodes` and `--relationships`, and a positional parameter, for example, in `--additional-config neo4j.properties --nodes 0-nodes.csv mydatabase`, the `--nodes` option behaves as greedy and the next option, in this case, `mydatabase`,  will be pulled in via the nodes convertor.
++
+See link:https://picocli.info/#_variable_arity_options_and_positional_parameters[Picocli -> Variable Arity Options and Positional Parameters] official documentation for more information.
+
+Therefore, either put the positional parameters first, e.g `mydatabase --nodes 0-nodes.csv`, or put the positional parameters last, after `--`, and after the final value of the last multi-value option, e.g `nodes 0-nodes.csv -- mydatabase`.
```

Comment:
```suggestion
Therefore, either put the positional parameters first, e.g `mydatabase --nodes 0-nodes.csv`, or put the positional parameters last -- after `--` and the final value of the last multi-value option, e.g `nodes 0-nodes.csv -- mydatabase`.
```

Reply:
Probably we can rewrite this sentence and make it as a list:
""""
To resolve the problem, you can:

- put the positional parameters first: for example, `mydatabase --nodes 0-nodes.csv`
- or put the positional parameters last, after `--` and the final value of the last multi-value option: for example, `nodes 0-nodes.csv -- mydatabase`.

""""